Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN dpkg --add-architecture i386 && \
 ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
 ENV PATH=$PATH:$JAVA_HOME
 
-RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py
+RUN curl https://bootstrap.pypa.io/pip/3.6/get-pip.py -o get-pip.py && python3 get-pip.py
```

Comment:
It was throwing an error saying that the get-pip.py script at that URL no longer supports 3.6, and to use this URL instead.